addons/website/views/snippets/s_banner.xml

```diff
@@ -18,8 +18,23 @@
                 <div class="o_grid_item o_grid_item_image g-col-lg-2 g-height-5 col-lg-2 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 3; grid-area: 2 / 11 / 7 / 13;">
                     <img class="img img-fluid mx-auto rounded" src="/web/image/website.s_banner_default_image_3" alt=""/>
                 </div>
-                <div class="o_grid_item o_grid_item_image g-col-lg-3 g-height-5 col-lg-3 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 4; grid-area: 7 / 7 / 12 / 10;">
-                    <img class="img img-fluid mx-auto rounded" src="/web/image/website.s_banner_default_image" alt=""/>
```

Comment:
Doing this way we'll keep `website .s_banner_default_image` defined, but we'll never use it 😉
Use it for the blockquote (L29 c.a.), eg:

```xml
<img src="/web/image/website.s_banner_default_image" class="s_blockquote_avatar img rounded-circle" alt=""/>
```

... and,  in `image_library.xml`, change the `url` attribute  of `s_banner_default_image` to the desired one -> `/website/static/src/img/snippets_demo/s_team_member_2.jpg`